packages/mds-policy-author/api.ts

```diff
@@ -100,21 +92,27 @@ function api(app: express.Express): express.Express {
         if (error instanceof AlreadyPublishedError) {
           return res.status(409).send({ error })
         }
-        return res.status(404).send({ error })
+        if (error instanceof NotFoundError) {
+          return res.status(404).send({ error })
+        }
+        if (error instanceof DependencyMissingError) {
```

Comment:
I don't think this error code is in the spec. Should go in there.

Reply:
Yup, I made a corresponding comment in the spec.